src/Modules/Settings/Dnn.PersonaBar.TaskScheduler/Services/TaskSchedulerController.cs

```diff
@@ -362,12 +362,13 @@ public HttpResponseMessage UpdateScheduleItem(ScheduleDto scheduleDto)
                 updatedItem.ScheduleID = scheduleDto.ScheduleID;
 
 
-                if ((updatedItem.ScheduleStartDate != existingItem.ScheduleStartDate) ||
-                    (updatedItem.Enabled) ||
-                    (updatedItem.TimeLapse != existingItem.TimeLapse) ||
-                    (updatedItem.RetryTimeLapse != existingItem.RetryTimeLapse) ||
-                    (updatedItem.RetryTimeLapseMeasurement != existingItem.RetryTimeLapseMeasurement) ||
-                    (updatedItem.TimeLapseMeasurement != existingItem.TimeLapseMeasurement))
+                if (updatedItem.ScheduleStartDate != existingItem.ScheduleStartDate ||
+                    updatedItem.Enabled ||
+                    updatedItem.Enabled != existingItem.Enabled ||
```

Comment:
Is it relevant to check whether the change is from disabled to enabled or the other way round?

Reply:
change from disabled to enabled should always do the normal update.